elastic/integrations
# [Security Solution] Update SentinelOne integration README with info. on configuring response actions

## Proposed commit message

- Adds information to the SentinelOne (S1) integration `README` file about the ability for sending Response Actions to S1 hosts
    - Changes goes along with Feature being implemented in Security Solution that supports sending `isolate` and `release` response actions.


## Checklist


## Author's Checklist

- none

## How to test this PR locally

- Start the stack using `--version=8.12.0-SNAPSHOT`:

```shell
elastic-package stack up -v -d --version=8.12.0-SNAPSHOT
```

- Navigate to the SentinelOne integration page
    - Note that the version at the top is `1.20.0`
    - Note the updated integration description that includes a link to Documentation
- Click the `documentation` link
    - You will (until v8.12.0 is GA'd) received a `404` "Page not found". This is expected since v8.12 has not yet been released. The page is accessible however by changing the `/current/` in the URL with `/master/`: <img width="912" alt="image" src="https://github.com/elastic/integrations/assets/56442535/bdcc0a80-7311-491e-81a1-43b84232dd6b">



## Related issues

- none

## Screenshots

<img width="1240" alt="image" src="https://github.com/elastic/integrations/assets/56442535/e84bfb9e-c508-429d-bbac-cf22fb02948b">
